Cinema system/Cinema system/Controllers/OrdersController.cs

```diff
@@ -22,27 +21,17 @@ public OrdersController(IOrderService orderService)
             _orderService = orderService;
         }
 
-        [HttpGet("{userId}")]
+        [HttpGet("users/{userId}/orders")]
         public IActionResult GetOrders(int userId)
         {
-            if (!_orderService.CheckUserExists(userId))
-            {
-                return NotFound(new Response(ErrorOfUserNonexistence));
-            }
-
```

Comment:
Are you sure this is the right solution? How is the fact that user is authenticated changes that he can pass invalid userId?
Btw here's another problem: if everyone can access this endpoint then every user can view every other user's orders. Is that expected behavior?